### DISCUSSION TITLE
How to customize the GUI with Qt studio?

### DISCUSSION THREAD

**Discussion by gladian2017** [OpenHantek/OpenHantek6022#379] [General]
Hi every body,
I have some experiences in editing GUI of applications from Qt studio enormous example repository to exactly what I am needing.  I need some  GUI customization for this project. How can I could perform that by Qt creator? I don't know even how to open the project in Qt creator since it lakes .pro file.
Thanks in advance for any help.

**Comment by Ho-Ro** [comment]
Just open the file `CMakeLists.txt` in the top directory. QtCreator will recognize this as the project file.
